### PR TITLE
Add 1.35 to travis build and tag 1.35 as latest in README

### PR DESCRIPTION
**Pull request by rosalieper**
See: https://phabricator.wikimedia.org/T264538

### CODE REVIEW

File: wikibase/1.31/.travis/build-deploy.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 #Oneline for full directory name see: https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 set -e
-docker build "$DIR/../base" -t wikibase/wikibase:1.31 -t wikibase/wikibase:1.31-base -t wikibase/wikibase:lts -t wikibase/wikibase:lts-base
-docker build "$DIR/../bundle" -t wikibase/wikibase:1.31-bundle -t wikibase/wikibase:lts-bundle
+docker build "$DIR/../base" -t wikibase/wikibase:1.31 -t wikibase/wikibase:1.31-base
+docker build "$DIR/../bundle" -t wikibase/wikibase:1.31-bundle
 
 if [ "$SHOULD_DOCKER_PUSH" = true ]; then
```
Comment (lucaswerkmeister) [inline wikibase/1.31/.travis/build-deploy.sh:8]: I’m not sure if I understand this correctly, but shouldn’t the three “lts” lines in this block move to the 1.35 file?

Comment (addshore) [reply wikibase/1.31/.travis/build-deploy.sh:8]: They seem to? (Not sure if I'm looking at a different version of the file / pr?)

Comment (lucaswerkmeister) [reply wikibase/1.31/.travis/build-deploy.sh:8]: The block _below_ my comment, within the SHOULD_DOCKER_PUSH, still has lts lines in the 1.31 file, unless I’m misreading the git diff.

Comment (addshore) [reply wikibase/1.31/.travis/build-deploy.sh:8]: +1 yes these should be removed!
(github UI made that hard to see)

image

**Review by addshore — CHANGES_REQUESTED** [review]

**Review by lucaswerkmeister — APPROVED** [review]
Looks good to me, but I don’t feel confident enough to merge.

File: wikibase/README.md [hunk 2]
```diff
@@ -9,12 +9,10 @@ Automated build.
 
 ### Tags
 
-Image name                                                            | Parent image
------------------------------------------------------------------     | ------------------------
-`wikibase/wikibase` : `latest`, `1.34`, `latest-base`, `1.34-base`    | [mediawiki:1.34](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `1.31`, `lts`, `lts-base`, `1.31-base`          | [mediawiki:1.31](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `latest-bundle`, `1.34-bundle`                  | [wikibase:1.34](https://hub.docker.com/r/wikibase/wikibase/)
-`wikibase/wikibase` : `lts-bundle`, `1.31-bundle`                     | [wikibase:1.31](https://hub.docker.com/r/wikibase/wikibase/)
+Image name                                                                               | Parent image
```
Comment (addshore) [inline wikibase/README.md:12]: This PR currently leaves 1.31 and 1.34 being built by travis.
If that is intended then these should remain in the docs here.
If that is not intended then they should be totally removed in this PR.

Looking at https://www.mediawiki.org/wiki/Version_lifecycle I think we want to keep 1.31 and 1.34 for now.
We will likely want to remove 1.34 at the end of this month.
1.31 will remain around until next year.

**Review by addshore — CHANGES_REQUESTED** [review]

File: wikibase/README.md [hunk 3]
```diff
@@ -9,12 +9,12 @@ Automated build.
...
+-------------------------------------------------------------------------------------    | ------------------------
+`wikibase/wikibase` : `latest`, `1.35`, `latest-base`, `1.35-base`, `lts`, `lts-base`    | [mediawiki:1.35](https://hub.docker.com/_/mediawiki/)
+`wikibase/wikibase` : `latest-bundle`, `1.35-bundle`, `lts-bundle`                       | [wikibase:1.35](https://hub.docker.com/r/wikibase/wikibase/)
+`wikibase/wikibase` : `1.34`, `1.34-base`                                                | [mediawiki:1.34](https://hub.docker.com/_/mediawiki/)
+`wikibase/wikibase` : `1.31`, `1.31-base`                                                | [mediawiki:1.31](https://hub.docker.com/_/mediawiki/)
```
Comment (addshore) [inline wikibase/README.md:17]: Looks like 1.31 and 1.34 are missing the bundle tags now (which are still being built)

**Review by addshore — APPROVED** [review]
Looks pretty good to me now, but I touched it last so someone else should do the squashing and merging!

**Comment by lucaswerkmeister** [comment]
Any idea why the 1.31 and 1.34 jobs of the push build failed?

**Comment by addshore** [comment]
I clicked the restart button, let's see what happens

**Comment by addshore** [comment]
All looks green now!

**Pull request merged**